src/components/dictionaryConcepts/containers/DictionaryConcepts.jsx

```diff
@@ -29,22 +34,110 @@ export class DictionaryConcepts extends Component {
       Diagnosis: PropTypes.string,
     })).isRequired,
     loading: PropTypes.bool.isRequired,
+    filterList: PropTypes.func.isRequired,
   };
 
   constructor(props) {
     super(props);
     this.state = {
       conceptsCount: getTotal(),
+      searchInput: '',
+      filterList: [],
     };
+    autoBind(this);
```

Comment:
maybe developer politics but I still think declaring this to apply to all functions might not be very effective, imagine a scenario where a developer does not know all the functions are bound by this statement. autobind('functionName1', 'functionName2, 'etc') is more recommended.

Reply:
You're right. But at the moment all the methods in the `class` in question need to be bound, If that changes in the future then we would consider binding the methods individually. Binding the methods individually now would defeat the purpose of using the `auto-bind` pkg. However, thank you for your feedback. @mwaz